examples/benchmark_problems/simple_tests/gauss_exact-fba.dat

```diff
@@ -0,0 +1,8 @@
+# FitBenchmark Problem
```

Comment:
Running this problem with `mantid` `Levenberg-Marquardt` gives very different results compared to the other format:
![results](https://user-images.githubusercontent.com/15614849/73061351-e1575b00-3e91-11ea-8c09-2d4e6032d955.png)

Reply:
I don't think there's anything we can do here -- must be a numerical issue due to the way the functions are handled.